scripts/release_scripts/run_macaron.sh

```diff
@@ -168,7 +161,7 @@ while [[ $# -gt 0 ]]; do
             ;;
         # This flag is duplicated for digest and database.
```

Comment:
If agreed, I would want to create a ticket for addressing this duplication too. A flag shouldn't be duplicated with different meaning across our tool.

Reply:
> If agreed, I would want to create a ticket for addressing this duplication too. A flag shouldn't be duplicated with different meaning across our tool.

There is a [ticket](https://github.com/oracle-samples/macaron/issues/211) to address this.